hack/verify-govet.sh

```diff
@@ -1,27 +1,26 @@
 #!/bin/bash
 
+set -o errexit
```

Comment:
Before we merge I want to fix the behavior on errors.

Because we `source "${STI_ROOT}/hack/util.sh"`, the script becomes effectively exit on error, and will exit on the first dir that fails `go vet`.